### ISSUE TITLE
Repeat marks incorrectly imported from Guitar Pro files

### ISSUE THREAD

**Issue by mylesdunlop** [powertab/powertabeditor#273]
Re Alpha 10 build for Windows
Repeat marks are incorrectly located following import of both gpx and gp5 files. It gets them both wrong, but wrong in a different way.

**Comment by cameronwhite** [comment]
Hi Myles, do you have a simple example file that demonstrates the issue? There has been a lot of work recently on the gpx importer so I'd like to see if this is still an issue.

**Comment by mylesdunlop** [comment]
@cameronwhite - Hi Cameron, thanks for following up. The issue seems to be fixed. 
It's been a while since I've done any tabbing, but I just downloaded and tried the Alpha 12 version on the original Powertab file I wrote. Now it generates the same repeat marks as does the Powertab 7.5 evaluation, which I just downloaded to check against. Using PowerTabEditor, I tried opening both *.gp and *.gpx versions of the file; both were correct.
